portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase

```diff
@@ -555,6 +555,2087 @@ definition {
 	}
 
 	@description = "LPS-134760: Given the Page Audit panel in a private page
+	@description = "LPS-137227 Given a content page
+	When a user with permissions to edit blogs visits the page
+	Then the issue icon for displaying the page audit panel shows"
+	@priority = "4"
+	test CheckPageAuditIconInContentPageWithEditBlogsPermission {
+		task ("Create user") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("Create role and assign to user") {
+			JSONRole.addRegularRole(
+				roleKey = "BlogsEditorRole",
+				roleTitle = "BlogsEditorRole");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "UPDATE",
+				resourceName = "com.liferay.blogs.model.BlogsEntry",
+				roleTitle = "BlogsEditorRole");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "BlogsEditorRole",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("Disable option to strangers to verify email") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "User Authentication",
+				configurationName = "General",
+				configurationScope = "Virtual Instance Scope");
+
+			PortalSettings.uncheckStrangersVerifyEmailCheckbox();
+		}
+
+		task ("Login with new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+		}
+
+		task ("Assert if Audit Page icon is present") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/content-page");
+
+			AssertElementPresent(locator1 = "PageAudit#PAGE_AUDIT_BUTTON");
+		}
+
+		task ("Login with Test user, to allow tearDown") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+		}
+
+		task ("Remove user and role created") {
+			JSONUser.tearDownNonAdminUsers();
+
+			JSONRole.deleteRole(roleTitle = "BlogsEditorRole");
+		}
```

Comment:
This should be included in the teardown, not in the test.
For the line _JSONUser.tearDownNonAdminUsers();_, you can just included it in the teardown.
For the line **JSONRole.deleteRole(roleTitle = "BlogsEditorRole");**, if you add the line _Role.tearDownCP();_ in the teardown, all roles with the word "Name" in the name of the role will be deleted, so I would include this line in the teardown and modify all the role names, to include the word "Name" :)